### PR TITLE
fix(components): use CSS only to determine whether to open arrow

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Arrow on disclosure doesn't flip in the live version

## Changes

Use CSS instead of conditional classes

### Fixed

- Arrow flips!

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

### CODE REVIEW

File: packages/components/src/Disclosure/Disclosure.css [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
   flex-basis: 0;
 }
 
-.flippedVertical {
+.details[open] .arrowIconWrapper {
   transform: rotateX(180deg);
 }
 
 .summary {
+  padding: var(--space-smaller) 0 var(--space-smallest) 0;
```
Comment (chris-at-jobber) [inline packages/components/src/Disclosure/Disclosure.css:12]: not related to bug, just a visual tweak while we are here

File: packages/components/src/Disclosure/Disclosure.css [hunk 2]
```diff
@@ -1,14 +1,15 @@
 .arrowIconWrapper {
-  transform-origin: 50% 50% 0;
+  transform-origin: 50% 47.5% 0;
```
Comment (chris-at-jobber) [inline packages/components/src/Disclosure/Disclosure.css:2]: Makes the arrow appear to flip on its' _visual_ y-axis center, not its' "real" y-axis center (arrows are weird optically)

**Comment by kingstonfung** [comment]
👍  thanks for catching & touching up this issue!

**Review by kingstonfung — APPROVED** [review]
👍

**Pull request merged**